docs/conf/prague/2021/schedule.rst

```diff
@@ -0,0 +1,201 @@
+:template: {{year}}/generic.html
+
+Schedule
+========
+
+Write the Docs is more than a conference.
+Each year we organize a wide range of events so that people can come together, collaborate, and learn from each other in different ways.
+
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: none
+
+
+{% if flaghashike or flaghasboat %}
+
+{{date.day_one.dotw}}, {{date.day_one.date}}
+--------------------------------------------------
+
+{% endif %}
+
+{% if flaghashike %}
+
+Hike
+~~~~
+
+The only event scheduled on Saturday is the :doc:`annual hike to Pittock Mansion </conf/{{shortcode}}/{{year}}/outing>`.
+If you get into town early, join us on the hike and take the chance to explore Portland in all of its glory.
+
+* **Where**: Lower Macleay Park or Macleay Park Entrance.
+* **When**: **{{ date.day_two.hike_time }} {{ tz }}**
```

Comment:
We already lost the changes I made here. I wonder if we're getting closer to the point we could make this an include, so we stop losing changes across versions?

Reply:
I will look into this tomorrow.

Reply:
I have extracted it into a generic schedule for 2021, taken from the Portland schedule you edited. Maybe we get stuck with that later, but I prefer that to copying and pasting and losing changes.